src/test/scala/uk/gov/hmrc/test/ui/pages/section5/RemoveItemsPage.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.test.ui.pages.section5
+
+import uk.gov.hmrc.test.ui.pages.base.BasePage
+import uk.gov.hmrc.test.ui.pages.base.Constants.yesNo
+import uk.gov.hmrc.test.ui.pages.section5.DetailsKeys._
+
+object RemoveItemsPage extends BasePage {
+
+  def backButtonHref: String = SummarySection5Page.path
+  val path: String = s"/declaration/remove-declaration-item/${details(ItemIds).head}"
+  val title: String = "Are you sure you want to remove declaration item 1?"
+
+  override def checkExpanders(): Unit = ()
+
+  // ex: fillPage()
+
+  override def fillPage(values: String*): Unit = if (selectYesOrNoRadio(values(yesNo))) clear(section5, details(ItemIds).headOption)
```

Comment:
Are you sure you always want to remove the `head` element? What if you want to remove the 2nd item?

Reply:
@udinobi  we wanted to check if we are able to remove an item. Either you remove first item or section item it will take you to the same page right.